### PR TITLE
Creating and adding TakeActionTips page.

### LINKED ISSUE #127: Split out Take Action Tips Into Its Own Page
There's a lot of info in the take action page, so we should **split this into a new page** and add a link instead, like "Own This Building? Take Action"

Screenshot_126

### PR DESCRIPTION
**Pull request by brayckner**
# Description

Created a page called TakeActionTips.vue, then removed the takeaways section from BuildingDetails.vue and added that into the new TakeActionTips page. Added a link under the "What Should We Do About This?" Header to navigate to the new Take Action Tips page. 

Fixes #127 

# Testing Instructions
Navigate to Building Details, scroll down to "What Should We Do About This?", then click the link that says "Own this Building? Take Action." It will then navigate you to the new page.

### CODE REVIEW

File: src/pages/TakeActionTips.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      </section>
+      </div>
+    </DefaultLayout>
+  </template>
+
+<script>
+export default {
+  name: 'TakeActionTips',
+};
+</script>
```
Comment (vkoves) [inline src/pages/TakeActionTips.vue:98]: Add a new line to the end of this file, just a good practice

File: src/pages/TakeActionTips.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<template>
+    <DefaultLayout>
+      <div class="layout-constrained">
+        <section class="takeaways">
+        <h2>What Should We Do About This?</h2>
```
Comment (vkoves) [inline src/pages/TakeActionTips.vue:5]: Can you switch this to an `<h1>` and add `id="main-content" and tabindex="-1"`? That matches other pages and will make it the "Skip to Main Content" link target for keyboard users.

File: src/pages/TakeActionTips.vue [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            </li>
+          </ul>
+        </p>
+      </section>
+      </div>
+    </DefaultLayout>
+  </template>
+
+<script>
+export default {
```
Comment (vkoves) [inline src/pages/TakeActionTips.vue:95]: I'd copy the script tag from another page so we can set the title, like this:

```ts


import { Component, Vue } from 'vue-property-decorator';

// TODO: Figure out a way to get metaInfo working without any
// https://github.com/xerebede/gridsome-starter-typescript/issues/37
@Component<any>({
  metaInfo() {
    return { title:  'Take Action' };
  },
})
export default class TakeAtionsTips extends Vue {
}
```

**Review by vkoves — CHANGES_REQUESTED** [review]
Looks great, just had a few minor comments to fix up a few thigns!

File: src/pages/TakeActionTips.vue (hunk 1 above)
Comment (brayckner) [reply src/pages/TakeActionTips.vue:98]: Fixed

File: src/pages/TakeActionTips.vue (hunk 2 above)
Comment (brayckner) [reply src/pages/TakeActionTips.vue:5]: Fixed

File: src/pages/TakeActionTips.vue (hunk 3 above)
Comment (brayckner) [reply src/pages/TakeActionTips.vue]: Fixed

File: src/pages/TakeActionTips.vue (hunk 2 above)
Comment (vkoves) [reply src/pages/TakeActionTips.vue:23]: @brayckner - I think you fixed it in the wrong place, this line of code hasn't changed! The takeaways heading was correct as an `h2`. I might just fix it real quick so we can merge this in, but wanted to mention

Comment (brayckner) [reply src/pages/TakeActionTips.vue]: I pushed up a correction to match the headings on the other pages. @vkoves

**Review by vkoves — APPROVED** [review]
Awesome work, thanks @brayckner! Confirmed the skip link works now and the page looks great!

**Comment by vkoves** [comment]
There's a few ESLint issues, but since this is a fork I'll just fix it on the `main` branch post merge

**Pull request merged**